tests/src/Kernel/fixtures/markup_rendering_patterns/copyright_overlay.yml

```diff
@@ -0,0 +1,54 @@
+copyright_overlay:
```

Comment:
you need to add this file to the list in the MarkupRenderingTest so it actually gets used

Reply:
added.